README.md

```diff
@@ -303,32 +306,34 @@ Meters report consumption over the service.
------|-------------------------------|------------|------------------------------------|-------------
-in   | cmd.meter.get_report          | string     | direction                          | Value - is a unit. May not be supported by all meters.
-in   | cmd.meter.reset               | null       |                                    | Resets all historical readings.
-out  | evt.meter.report              | float      | unit, prv_data, delta_t, direction |
-out  | evt.meter_ext.report          | float_map  |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
-in   | cmd.meter_ext.get_report      | null       |                                    | Request extended report
+Dir  | `type`                        | `val_t`            | `prop`                             | Description
+-----|-------------------------------|--------------------|------------------------------------|-------------
+in   | cmd.meter.get_report          | string             | direction                          | Triggers evt.meter.report. An argument is "export" or "import".
+in   | cmd.meter.reset               | null               |                                    | Resets all historical readings.
+out  | evt.meter.report              | float              | unit, prv_data, delta_t, direction | Meter reading
+out  | evt.meter_ext.report          | float_map          |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
+in   | cmd.meter_ext.get_report      | null               |                                    | Request extended report
 
 #### Interface props
 
-Name         | Value example | Description
--------------|---------------|-------------
-`delta_t`    |               | Elapsed time in seconds between the 'Meter Value' and the 'Previous Meter Value' measurements.
-`prv_data`   |               | Previous meter reading.
-`unit`       | "kWh"         | One of sup_units. For meter_unknown it will be a number.
-`direction`  | "export"      | Defines whether reading is based on consumption (import) or production (export). Applicable for every meter except for meter_elec.
+Name         | Value type                  | Description
+-------------|-----------------------------|-------------
+`delta_t`    | int [seconds]               | Time between the current and the previous meter reading.
+`prv_data`   | float                       | Previous meter reading.
+`unit`       | Defined by prop `sup_units` | Unit of a value in `val` field.
+`prv_unit`   | Defined by prop `sup_units` | Unit of a value in `prv_data` prop.
+`direction`  | string                      | Defines whether reading is based on consumption ("import") or production ("export").
+`virtual`    | null                        | Informs that energy measurement was calculated by Virtual energy service. 
```

Comment:
Guys, important thing. Properties ARE ALWAYS strings. It's always map of string keys and string values. Even if it is integer it should be returned as "11" for instance. @BudzikM Please do not change the layout of documentation that deviates from existing ones. The column should contain example values.

Also, i'd make virtual value to be "true".